app/controllers/async_info_controller.rb

```diff
@@ -56,6 +56,7 @@ def user_data
         id: @user.id,
         name: @user.name,
         username: @user.username,
+        email: @user.email,
```

Comment:
We want this to populate text in the liquid tag. Something like:
> Hey, user! Are you sure you want to share your email address (test@test.com) with the author_username?

The exact wording is TBD, but we'll want the email address to it's as clear as possible to the user what they're agreeing to.